src/components/pages/ParentFamily/ParentFamilyHome.js

```diff
@@ -82,29 +83,24 @@ const ParentFamilyHome = props => {
             </Card>
           </Col>
 
-          <Col span={8}>
-            <Card style={{ width: 300 }} className="student-card">
-              <Meta
-                avatar={<Avatar src="https://joeschmoe.io/api/v1/random" />} // make dynamic with state management
-                title="Student Name"
-              />
-              <Button className="student-view-account-button">
-                VIEW ACCOUNT
-              </Button>
-            </Card>
-          </Col>
-
-          <Col span={8}>
-            <Card style={{ width: 300 }} className="student-card">
-              <Meta
-                avatar={<Avatar src="https://joeschmoe.io/api/v1/random" />} // make dynamic with state management
-                title="Student Name"
-              />
-              <Button className="student-view-account-button">
-                VIEW ACCOUNT
-              </Button>
-            </Card>
-          </Col>
+          {children &&
+            children.map(child => {
+              return (
+                <Col span={8} key={child.child_id}>
+                  <Card style={{ width: 300 }} className="student-card">
+                    <Meta
+                      avatar={
+                        <Avatar src="https://joeschmoe.io/api/v1/random" />
+                      } // avatar url for student not in backend
+                      title={child.username}
+                    />
+                    <Button className="student-view-account-button">
+                      VIEW ACCOUNT
+                    </Button>
+                  </Card>
+                </Col>
+              );
+            })}
         </Row>
```

Comment:
Great implementing state management here for the children and for the message of why avatar URL is missing.